lmdeploy/lite/apis/kv_qparams.py

```diff
@@ -101,18 +101,17 @@ def main(model: str,
          num_tp: int = 1,
          calib_dataset: str = 'c4',
          calib_samples: int = 128,
-         output_dir: str = './kv_scales'):
+         output_dir: str = './kv_scales',
+         device: str = 'cuda'):
     assert granularity in ['per_tensor'], \
         'Currently, only support per-tensor quantization for the kv cache.'
     assert bits == 8, \
         'Currently, only support 8-bit quantization for the kv cache.'
     assert calib_dataset in ['c4', 'ptb', 'wikitext2', 'pileval'], \
         'Currently, only support `c4`, `ptb`, `wikitext2`, or `pileval`.'
 
-    tokenizer = AutoTokenizer.from_pretrained(model,
-                                              use_fast=False,
-                                              trust_remote_code=True)
-    model = AutoModelForCausalLM.from_pretrained(model, trust_remote_code=True)
+    tokenizer = AutoTokenizer.from_pretrained(model, use_fast=False)
+    model = AutoModelForCausalLM.from_pretrained(model)
```

Comment:
Why remove the `trust_remote_code=True`?

Reply:
My bad, there seem to be some issues when rebasing.